### PR TITLE
Make ExecuteCommand create graphs too

### PR DESCRIPTION
**Pull request by tuxji**
Andy, Abha asked me to make ExecuteCommand create graphs too.  These changes should be merged into the development branch after you review them.

Previously ExecuteCommand would run the inference processor but not the run inference handler which also displays the results and creates the graphs in addition to running the inference processor.  Now ExecuteCommand will hand the entire job off to the run inference handler instead of the run inference processor.

The key part to making these changes work is adding new Guice bindings to SADLCliAppRuntimeModule so that the run inference handler will be able to print messages, find project directories, and create graphs successfully.

We also need the com.ge.research.sadl.ide bundle to export com.ge.research.ide.utils so SADLCliAppRuntimeModule can import SadlIdeProjectHelper from it.

Also remove -console from ExecuteCommand.sh and ExecuteCommand.cmd scripts since it just produces an osgi> prompt that we don't need in the output.  Also improve log4j.properties in case we want to use it (ExecuteCommand also configures logging programmatically anyway).

Update SadlConfiguration.xml (and ExecuteCommand) to include new SADL properties that were added to SADL recently.

Reformat ExecuteCommand and rest of its package classes using Google Java Formatter.

Make GraphVizVisualizer, SadlActionHandler, and ExecuteCommand use SL4J Logger API to ensure their logging will be handled appropriately.

Note that more than half of SADL uses SLF4J Logger but rest of SADL still uses Log4j Logger and probably should be converted to use SLF4J so that all logging will end up in one place.

### CODE REVIEW

**Comment by crapo** [comment]
@kittaakos , please review to see if any of these changes will pose a problem for the Web version.

File: sadl3/com.ge.research.sadl.parent/com.ge.research.sadl.ui/src/com/ge/research/sadl/ui/handlers/SadlActionHandler.java [hunk 1]
```diff
@@ -73,10 +73,13 @@
 import com.ge.research.sadl.utils.SadlConsole;
 import com.google.inject.Inject;
 import com.google.inject.Injector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @SuppressWarnings("restriction")
 public abstract class SadlActionHandler extends AbstractHandler {
 
+	private final Logger logger = LoggerFactory.getLogger(SadlActionHandler.class);
```
Comment (kittaakos) [inline sadl3/com.ge.research.sadl.parent/com.ge.research.sadl.ui/src/com/ge/research/sadl/ui/handlers/SadlActionHandler.java]: [minor]: Please use `private static final`.

**Comment by kittaakos** [comment]
screen shot 2019-02-07 at 11 58 35

Just a note, you are going to merge it into the `master`. Usually, it should be `development`.

**Comment by tuxji** [comment]
Good catch!  Yes, I'd wanted to merge the changes into development, not master.  Andy, we could avoid mistakes like these by telling GitHub to set the default branch to be development instead of master (that also will mean that a git clone won't need to be followed by a git checkout development).  I've corrected the base branch, made the static final LOGGER change and run mvn clean install to check for compilation errors.  You will be able to merge the pull request after the continuous integration checks pass.

**Comment by kittaakos** [comment]
> telling GitHub to set the default branch to be development instead of master (that also will mean that a git clone won't need to be followed by a git checkout development)

💯 👍 

> made the static final LOGGER

Thank you!

**Comment by crapo** [comment]
I have changed the default branch to development--it came with dire warnings.

**Pull request merged**